src/components/EditSection.tsx

```diff
@@ -46,40 +53,39 @@ export const EditSection: React.FunctionComponent<EditSectionProps> = ({ page })
                 <span className="text-muted mx-1" aria-hidden="true">
                     •
                 </span>{' '}
-                <Link href={`https://github.com/sourcegraph/handbook/commits/main/${CONTENT_FOLDER}/${page.path}`}>
+                <Link
+                    href={`https://github.com/sourcegraph/handbook/commits/main/${CONTENT_FOLDER}/${page.path.replace(
+                        /^\//,
+                        ''
+                    )}`}
+                >
                     History
                 </Link>
             </div>
         )}
         <div className="sidebar-bottom-links">
-            {page.path === 'company/team/index.md' && (
-                <Link href="https://github.com/sourcegraph/handbook/edit/main/data/team.yml">
-                    Edit this data on GitHub
-                </Link>
-            )}
-            {page.path === 'product/product_areas.md' && (
-                <Link href="https://github.com/sourcegraph/handbook/edit/main/data/product_areas.yml">
-                    Edit this data on GitHub
-                </Link>
-            )}
-            {page.path === 'product/feature_maturity.md' && (
-                <Link href="https://github.com/sourcegraph/handbook/edit/main/data/features.yml">
-                    Edit this data on GitHub
-                </Link>
-            )}
-            {page.path === 'product/feature_compatibility.md' && (
-                <Link href="https://github.com/sourcegraph/handbook/edit/main/data/features.yml">
-                    Edit this data on GitHub
+            {Object.keys(GENERATED_PAGE_DATA).map(pagePath => {
```

Comment:
This will help you get around the error:
```suggestion
            {Object.entries(GENERATED_PAGE_DATA).map(([pagePath, yamlFile]) => {
```